### ISSUE TITLE
SQLException on ActionDao#getLatestActions when the table is empty

### ISSUE THREAD

**Issue by manuzhang** [Intel-bigdata/SSM#565]
```
[ERROR] [07/04/2017 09:38:35.587] [DruidPooledPreparedStatement] getFetchDirection error
java.sql.SQLException: ResultSet closed
	at org.sqlite.core.CoreResultSet.checkOpen(CoreResultSet.java:69)
	at org.sqlite.jdbc3.JDBC3ResultSet.getFetchDirection(JDBC3ResultSet.java:116)
	at org.sqlite.jdbc3.JDBC3Statement.getFetchDirection(JDBC3Statement.java:324)
	at com.alibaba.druid.filter.FilterChainImpl.statement_getFetchDirection(FilterChainImpl.java:2537)
	at com.alibaba.druid.filter.FilterAdapter.statement_getFetchDirection(FilterAdapter.java:2514)
	at com.alibaba.druid.filter.FilterChainImpl.statement_getFetchDirection(FilterChainImpl.java:2535)
	at com.alibaba.druid.proxy.jdbc.StatementProxyImpl.getFetchDirection(StatementProxyImpl.java:283)
	at com.alibaba.druid.pool.DruidPooledPreparedStatement.<init>(DruidPooledPreparedStatement.java:98)
	at com.alibaba.druid.pool.DruidPooledConnection.prepareStatement(DruidPooledConnection.java:355)
	at org.springframework.jdbc.core.JdbcTemplate$SimplePreparedStatementCreator.createPreparedStatement(JdbcTemplate.java:1521)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:627)
	at org.springframework.jdbc.core.JdbcTemplate.query(JdbcTemplate.java:684)
	at org.springframework.jdbc.core.JdbcTemplate.query(JdbcTemplate.java:716)
	at org.springframework.jdbc.core.JdbcTemplate.query(JdbcTemplate.java:726)
	at org.springframework.jdbc.core.JdbcTemplate.query(JdbcTemplate.java:776)
	at org.smartdata.metastore.dao.ActionDao.getLatestActions(ActionDao.java:95)
	at org.smartdata.metastore.MetaStore.getNewCreatedActionsTableItem(MetaStore.java:640)
	at org.smartdata.server.engine.CmdletManager.listNewCreatedActions(CmdletManager.java:370)
```

Note there will be no exception when the `limit` statement is removed from the underlying sql clause. (That's why `RuleDao#getAll()` doesn't error)

```sql
select * from actions 
   WHERE finished = 1 
   ORDER by create_time 
   DESC 
   limit 20
```

**Comment by qiyuangong** [comment]
Thanks @manuzhang .
In sqlite, an empty select query with limit will fail when database is empty. In mysql, this query will return an empty list.

**Comment by manuzhang** [comment]
Fixed in #569
